inc/upgrade/namespace.php

```diff
@@ -83,3 +87,17 @@ function upgrade_database_3() {
 
 	$wpdb->query( $query );
 }
+
+/**
+ * Upgrade Cavalcade database tables to version 4.
+ *
+ * Remove nextrun index as it negatively affects performance.
+ */
+function upgrade_database_4() {
```

Comment:
Can we also update the v3 alter command so that someone migrating from v2 -> v4 doesn't get a useless index in the meantime?

Reply:
@rmccue like that?